src/SynthImage/distort_augmentation.py

```diff
@@ -0,0 +1,103 @@
+from enum import Enum
+
+import cv2
+import numpy as np
+from PIL import Image
+
+
+class DistortionMode(Enum):
+    """A simple selection for the mode used for font contour distortion"""
+
+    additive = 0
+    subtractive = 1
+
+
+def distort_line(
```

Comment:
I would define the function under apply_distort because of the same reason i have mentioned for the deform method.